Source/Shaders/GlobeFS.glsl

```diff
@@ -67,6 +71,7 @@ varying vec3 v_mieColor;
 vec4 sampleAndBlend(
     vec4 previousColor,
     sampler2D texture,
+    sampler2D textureColorPalette,
```

Comment:
Since you don have the same number of color palettes, create a uniform of indices into `u_dayTextureColorPalette` for each texture.